app/services/follow_service.rb

```diff
@@ -8,15 +8,15 @@ def follow(entity, user, reason)
       return if (reason == 'created' || reason == 'contributed') && !should_auto_follow(entity, user, reason)
 
       follow = Follow.where(followed: entity, user: user).first_or_create
-      follow.update_attribute('muted', false)
+      follow.update(muted: false)
```

Comment:
```
unless follow.update(muted: false)
  # not sure what to do here...
  # return false?
  # fail follow.errors.messages?
  # or something else
end
```

Reply:
I'm not sure either, is it horrible to just a leave a TODO?

Reply:
I prefer throwing an exception so well get notified